sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -580,6 +578,16 @@ static bool isOnlineLinkEnabled();
 // End forward declarations for mock Level Zero APIs
 std::once_flag OnceFlag;
 
+// This function will ensure compatibility with both Linux and Windowns for
+// setting environment variables.
+static void setEnvVar(const char *name, const char *value) {
```

Comment:
I suggest checking error-code and return bool here, indicating if var was set, even if currently you aren't going to use it.